quartermaster/src/main/java/com/github/reflectoring/infiniboard/quartermaster/dashboard/rest/DashboardResourceAssembler.java

```diff
@@ -0,0 +1,28 @@
+package com.github.reflectoring.infiniboard.quartermaster.dashboard.rest;
+
+import org.springframework.hateoas.mvc.ResourceAssemblerSupport;
+import org.springframework.stereotype.Component;
+
+import com.github.reflectoring.infiniboard.quartermaster.dashboard.domain.Dashboard;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
+
+@Component
+public class DashboardResourceAssembler extends ResourceAssemblerSupport<Dashboard, DashboardResource> {
+
+    public DashboardResourceAssembler() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)